CODEOWNERS

```diff
@@ -155,3 +155,4 @@ k8s/**/common/bsp/namespace.yaml @hmcts/platform-engineering @hmcts/bulk-scan
 k8s/**/common/bsp/identity.yaml @hmcts/platform-engineering @hmcts/bulk-scan
 k8s/aat/common/rpe/bulk-scan-payment-processor.yaml @hmcts/platform-engineering @hmcts/bulk-scan
 k8s/**/common/bsp/bulk-scan-processor.yaml @hmcts/platform-engineering @hmcts/bulk-scan
+k8s/**/common/bsp/bulk-scan-sample-app.yaml @hmcts/platform-engineering @hmcts/bulk-scan
```

Comment:
```suggestion
k8s/aat/common/bsp/bulk-scan-sample-app.yaml @hmcts/platform-engineering @hmcts/bulk-scan
```

Reply:
ohh yes good point also this app is only to be deployed to AAT.